avm/res/synapse/workspace/main.bicep

```
@@ -31,6 +31,10 @@ param defaultDataLakeStorageFilesystem string
 @description('Optional. Create managed private endpoint to the default storage account or not. If Yes is selected, a managed private endpoint connection request is sent to the workspace\'s primary Data Lake Storage Gen2 account for Spark pools to access data. This must be approved by an owner of the storage account.')
 param defaultDataLakeStorageCreateManagedPrivateEndpoint bool = false
 
+import { adminType } from 'administrators/main.bicep'
+@description('Optional. The Entra ID administrator for the synapse workspace.')
+param administrator adminType = { administratorType: '', login: '', sid: '' }
```

Do I understand this right, that the default is empty - and if it's empty it's not supposed to be deployed? 

If the case, you can just make this parameter nullable and check for that :)
```suggestion
param administrator adminType?
```